controllers/articlesController.js

```diff
@@ -49,16 +49,31 @@ export default class ArticlesController {
    */
   static async getAll(req, res) {
```

Comment:
#### Nitpick
- Can you refactor this to something like `getAllArticles`, thank you.

Reply:
Thank you @aanchirinah, I will refactor the code.